packages/web-components/src/components/va-header-minimal/va-header-minimal.tsx

```diff
@@ -68,8 +73,12 @@ export class VaHeaderMinimal {
             <img class="va-logo" src={vaSeal} alt="VA logo and Seal, U.S. Department of Veterans Affairs" />
           </a>
           <div class="header-container">
-            <h1>{header}</h1>
-            {this.subheader && <h2>{subheader}</h2>}
+            
+            {!disableHeadings ? <h1>{header}</h1> : <div class="header">{header}</div>}
+
+            {subheader && (
+              !disableHeadings ? <h2>{subheader}</h2> : <div class="subheader">{subheader}</div>
```

Comment:
@rsmithadhoc I think this all looks great to me and thank you for doing the work! 

Quick question about using `div`; is it possible that a team would need to make this a header but not an `h1`? (h2, h3, etc)? We've had this come up before with other components so just double checking.

Reply:
@jamigibbs For this case, I don't think they would need headings other than `<h1>` followed by `<h2>`. If they use this header with the headings, it should be at the top of the page and it should contain the page's title. The issue of using another heading level might still come up, but I think a heading other than `<h1>` would be an improper usage of the component that we shouldn't allow.